src/main/java/BinarySearch.java

```diff
@@ -4,7 +4,7 @@
 
     public static void main(String[] args) {
         int[] sortedArray = {1, 2, 3, 4, 5, 6, 7, 8, 9};
-        int key = 8;
+        int key = 9;
```

Comment:
why is there a single change in another file as part of this PR?

Reply:
I was playing around with the binarysearch method to make sure I knew how it worked. Forgot to reset it back to original value.